NoteFind/Alerts/ErrorAlerts.m

```diff
@@ -108,4 +108,26 @@ + (void)errorPostingNote:(NewNoteViewController *)newNoteVC {
         
     }];
 }
+
++ (void)errorDownloading:(UIViewController *)vc {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"error downloading note" message:@"there was a problem trying download your note. please try again" preferredStyle:(UIAlertControllerStyleAlert)];
+
+    UIAlertAction *okAction = [UIAlertAction actionWithTitle:@"ok" style:UIAlertActionStyleDefault handler:^(UIAlertAction * _Nonnull action) { /*handle response here */ }];
+    [alert addAction:okAction];
+    
+    [vc presentViewController:alert animated:YES completion:^{
+        
+    }];
+}
+
++ (void)successDownloading:(UIViewController *)vc {
+    UIAlertController *alert = [UIAlertController alertControllerWithTitle:@"success!" message:@"this note was successfully downloaded! check your profile for all of your downloaded notes." preferredStyle:(UIAlertControllerStyleAlert)];
```

Comment:
This can be in another PR but do you think it might make sense to create a helper function for all these dialogs since the code looks pretty identical for each one?

Reply:
definitely! i'll include this on another PR